modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +42,22 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def attempt_submit(bgs_claim)
+          response = bgs_service.benefit_claims.update_bnft_claim(claim: bgs_claim)
+
+          if !response[:bnft_claim_dto].nil?
```

Comment:
why `if !` instead of `unless` ? Curious, this could be a rubocop thing, just checking.

Also could use `.dig` here as well just in case.